packages/artblocks/src/abi.ts

```diff
@@ -0,0 +1,698 @@
+const PURCHASE_SET_PRICE_V5_ABI = [
```

Comment:
Any contracts that do not have their own ABI are already covered by an existing ABI.